library/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation 'com.squareup.retrofit2:converter-gson:2.5.0'
 
     implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'com.aminography:redirectglide:1.0.3'
```

Comment:
Why it's added? Can't see any connection with notification manager task.

Reply:
it was added for showing image in notification (old variant of implementation). I forgotten delete it.

Reply:
@andriizhumela  comment was resolved by the dependency is still there

Reply:
in this implementation we need this for loading image. Server return url with redirection.